### PR TITLE
Update el8 images

### PR DESCRIPTION
**Pull request by mmascher**
See https://its.cern.ch/jira/browse/CMSPROD-65 for details

### CODE REVIEW

**Comment by mmascher** [comment]
See https://its.cern.ch/jira/browse/CMSPROD-65 for details

File: cms/tags.yaml [hunk 1]
```diff
@@ -6,5 +6,4 @@ rhel7-itb: rhel7-m20220512
 rhel8-itb: cmssw/el8:x86_64
 rhel6: rhel6-m20211005
 rhel7: rhel7-m20220331
-rhel8: rhel8-m-m20220516
-rhel8-m: rhel8-m-m20221009
+rhel8: cmssw/el8:x86_64
```
Comment (smuzaffar) [inline cms/tags.yaml:9]: @mmascher , do you really want to go with `cmssw/el8:x86_64` moving target ? Note that `cmssw/el8:x86_64` automatically gets rebuild every time the base image (almalinux:8) is updated or if we have added a new/remove any package.

Comment (mmascher) [reply cms/tags.yaml:9]: Ah no, sorry, I misunderstood in the ticket. Can we create a tag or something? Also, please wait until Monday before merging, please :)

Comment (mmascher) [reply cms/tags.yaml:9]: Just read the ticket again. I will use `cmssw/el8:x86_64-d20231016`

Comment (smuzaffar) [reply cms/tags.yaml]: the latest tag (as mentioned in the Jira ticket) is `cmssw/el8:x86_64-d20231016`  which was built on 16th OCT

**Comment by smuzaffar** [comment]
ok, we will merge it on Monday then

**Comment by mmascher** [comment]
@smuzaffar Let's merge this!

**Comment by smuzaffar** [comment]
@mmascher , we have re-tagged and deployed following [a] new images under `/cvmfs/singularity.opensciencegrid.org`. The new images allows us to use `x86_64, aarch64, and ppc64le` from `/cvmfs/singularity.opensciencegrid.org`. New images names are same as the old images but just contain `-$(uname -m)` suffix. Extra arch based suffix was needed to get these images deployed on the cvmfs.

Can we start testing the new images on ITB and may be more the productions jobs to use these images?
Thanks,
--Shahzad

[a] 
- ITB images
  - x86_64
    - cmssw/cms:rhel6-itb-x86_64  (same as cmssw/cms:rhel6-itb)
    - cmssw/cms:rhel7-itb-x86_64  (same as cmssw/cms:rhel7-itb)
    - cmssw/cms:rhel8-itb-x86_64  (same as cmssw/cms:rhel8-itb)
    - cmssw/cms:rhel9-itb-x86_64  (same as cmssw/cms:rhel9-itb)
  - aarch64
    - cmssw/cms:rhel8-itb-aarch64
    - cmssw/cms:rhel9-itb-aarch64
  - ppc64le
    - cmssw/cms:rhel8-itb-ppc64le
- Production
  - x86_64
    - cmssw/cms:rhel6-x86_64  (same as cmssw/cms:rhel6)
    - cmssw/cms:rhel7-x86_64  (same as cmssw/cms:rhel7)
    - cmssw/cms:rhel8-x86_64  (same as cmssw/cms:rhel8)
    - cmssw/cms:rhel9-x86_64  (same as cmssw/cms:rhel9)
  - aarch64
    - cmssw/cms:rhel8-aarch64
    - cmssw/cms:rhel9-aarch64
  - ppc64le
    - cmssw/cms:rhel8-ppc64le

**Pull request merged**